php-pre-challenge1/kadai1.php

```diff
@@ -0,0 +1,12 @@
+<?php
+for ($i =1; $i <= 100; $i++){
+        if ($i % 15 == 0){
```

Comment:
```suggestion
        if ($i % 15 === 0){
```
厳密比較 `===` にしてください。
どちらでも動く場合が多いですが、昨今は `===` だけを使うようにするのがおすすめです。

Reply:
厳密比較にしました！